package.json

```diff
@@ -13,7 +13,7 @@
     "lint": "run-p lint-js lint-css",
     "lint-js": "eslint *.js bin src",
     "lint-fix-js": "yarn lint-js --fix",
-    "lint-css": "stylelint 'src/**/*.css' 'res/css/**/*.css'",
+    "lint-css": "stylelint 'src/**/*.css' 'res/**/*.css'",
```

Comment:
I realized that the css for the photon example page wasn't checked. I did it in the first commit that you might want to look separately.

Reply:
Ah, whoops! Thanks for the fix here.